### ISSUE TITLE
Lower acc of resnet50 than torchvision

### ISSUE THREAD

**Issue by leoozy** [D-X-Y/ResNeXt-DenseNet#23]
Hi, I trained the resnet50 for 90 epoch, the top1 acc is 75.61%. The top-1 acc for corresponding pretrained model on torchvison is 76.15.  Is this caused by different epochs or some mistakes since I do not find how many epochs are used for torchvison.

**Comment by D-X-Y** [comment]
TorchVision trained the model for 100 epochs with the distributed training, which could obtain slightly better performance than this repo.
I would like you to use my new repo at here: https://github.com/D-X-Y/NAS-Projects/blob/master/BASELINE.md#performance-on-imagenet , in which the trained resnet-50 can achieve 77.6% top-1 accuracy.
